apps/trading/.env

```diff
@@ -19,6 +19,7 @@ NX_DEPLOY_URL=$DEPLOY_URL
 NX_DEPLOY_PRIME_URL=$DEPLOY_PRIME_URL
 
 # App configuration variables
+NX_VEGA_ENV = "TESTNET"
 NX_VEGA_URL = "https://lb.testnet.vega.xyz/query"
 NX_ETHEREUM_CHAIN_ID = 3
-NX_INFURA_ID = "4f846e79e13f44d1b51bbd7ed9edefb8"
\ No newline at end of file
+NX_INFURA_ID = "4f846e79e13f44d1b51bbd7ed9edefb8"
```

Comment:
chore (blocking): let's not tie ourselves explicitly to infura. Can we make this NX_ETH_PROVIDER_URL

Reply:
Reason being is that once we move to capsule this will be a local running node and so infura won't know anything about it